bind.go

```diff
@@ -561,6 +562,11 @@ func bindSTDFlag(fs STDFlagSet, tag flagTag, p interface{}) bool {
 	case *string:
 		val := *p
 		fs.StringVar(p, tag.Name, val, tag.Usage)
+	case textBidiMarshaler:
+		// Match the interface after concrete types so that any concrete types that
+		// also implement the interface use the more specific implementation for
+		// their concrete types.
```

Comment:
Great consideration.

Reply:
Found based on existing tests, but making the rationale clear in a comment seemed like it'd be helpful